src/main/java/uk/gov/companieshouse/psc/delta/service/api/ApiClientServiceImpl.java

```diff
@@ -0,0 +1,77 @@
+package uk.gov.companieshouse.psc.delta.service.api;
+
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.api.InternalApiClient;
+import uk.gov.companieshouse.api.http.ApiKeyHttpClient;
+import uk.gov.companieshouse.api.http.HttpClient;
+import uk.gov.companieshouse.api.model.ApiResponse;
+import uk.gov.companieshouse.api.psc.FullRecordCompanyPSCApi;
+import uk.gov.companieshouse.logging.Logger;
+
+
+/**
+ * Service that sends REST requests via private SDK.
+ */
+@Primary
+@Service
+public class ApiClientServiceImpl extends BaseApiClientServiceImpl implements ApiClientService {
+
+    @Value("${api.pscFullRecord-data-api-key}")
```

Comment:
I think you just need this to be `api.psc-data-api-key` the consumer will only ever use one key